src/test/java/seedu/address/logic/parser/FindCommandParserTest.java

```diff
@@ -20,15 +26,67 @@ public void parse_emptyArg_throwsParseException() {
         assertParseFailure(parser, "     ", String.format(MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
     }
 
+    @Test
+    public void parse_moreThanOnePrefix_throwsParseException() {
+        assertParseFailure(parser, " n/CS1231S Homework mod/CS2100", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MORE_THAN_ONE_PREFIX_MESSAGE));
+    }
+
+    @Test
+    public void parse_invalidArgs_throwsParseException() {
+        //invalid prefix
+        assertParseFailure(parser, " boo/CS2100", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.MESSAGE_USAGE));
+        //invalid keywords depending on prefix
+        assertParseFailure(parser, " d/2345 hello", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.INVALID_DATE_OR_TIME_MESSAGE));
+
+        assertParseFailure(parser, " d/99-99-1010 2500", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.INVALID_DATE_OR_TIME_MESSAGE));
+
+        assertParseFailure(parser, " d/99-10-2020", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.INVALID_DATE_OR_TIME_MESSAGE));
+
+        assertParseFailure(parser, " d/9800", String.format(
+                MESSAGE_INVALID_COMMAND_FORMAT, FindCommand.INVALID_DATE_OR_TIME_MESSAGE));
+
+        assertParseFailure(parser, " mod/CS386", ModuleCode.MESSAGE_CONSTRAINTS);
+
+        assertParseFailure(parser, " priority/NO", Priority.MESSAGE_CONSTRAINTS);
```

Comment:
You may consider adding in comments to tell the reader exactly what you are testing for (e.g invalid deadline, invalid date, valid time or invalid module code). Though this is just a minor nitpick :)

Reply:
Amended!